src/browser/components/JobDescription.styl

```diff
@@ -0,0 +1,15 @@
+.job
+  h1
+    font-size 2.5em
+    text-transform uppercase
```

Comment:
zly pattern. Tutaj mozesz zalozyc klase i ja uzywac w kilku miejsca bo robisz to samo. DRY.